ReCaptchaFrontendUi/view/frontend/web/js/reCaptcha.js

```diff
@@ -11,9 +11,10 @@ define(
         'jquery',
         'ko',
         'Magento_ReCaptchaFrontendUi/js/registry',
-        'Magento_ReCaptchaFrontendUi/js/reCaptchaScriptLoader'
+        'Magento_ReCaptchaFrontendUi/js/reCaptchaScriptLoader',
+        'Magento_ReCaptchaFrontendUi/js/nonInlineReCaptchaRenderer',
     ],
-    function (Component, $, ko, registry, reCaptchaLoader, undefined) {
+    function (Component, $, ko, registry, reCaptchaLoader,nonInlineReCaptchaRenderer, undefined) {
```

Comment:
We don't need to pass `undefined`. Undefined it's system type, we shouldn't use it directly. Any variable/property that is not defined is `undefined` type by default. If we want to reset variable/property value we should use `null`, but in this case, it's no needed at all

Reply:
Updated